bin/nwbuild

```diff
@@ -9,7 +9,6 @@ var argv = optimist
     .usage('Usage: nwbuild [options] [path]')
 
     .alias('p', 'platforms')
-    .default('p', 'osx32,osx64,win32,win64')
```

Comment:
Why not just remove `osx32` here?

Reply:
i'am not share if someone using the old version node on osx32 system.

Reply:
I mean you're moving the default options somewhere else. Wouldn't it work the same if you changed this line to `.default('p', 'osx32,osx64,win32,win64')`?

Reply:
I agree with @adam-lynch, removing `osx32` here should be equivalent (unless I'm missing something).

PS: @adam-lynch you mean `.default('p', 'osx64,win32,win64')`, right?